sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -494,11 +494,7 @@ def run_pipeline(self, pipeline, options):
       if pre_optimize == 'none' or pre_optimize == 'default':
         phases = []
       elif pre_optimize == 'all':
-        phases = [
-            # TODO(BEAM-11694): Enable translations.pack_combiners
-            # translations.pack_combiners,
-            translations.sort_stages
-        ]
+        phases = [translations.pack_combiners, translations.sort_stages]
```

Comment:
Is it safe to enable combiner packing for Dataflow now, or do we need to wait for Dataflow's backend to support CombinePerKeys without subtransforms first?

Reply:
The goal is that it will by the time this release actually goes out.